src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,51 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+public class FindCommand extends Command {
+    private String search;
```

Comment:
Overall good variable naming, but for this case, maybe you can consider using a more descriptive variable name for this String? For example, "toSearch", "keyword", "toFind".